src/Kernel/Delay.class.st

```diff
@@ -60,12 +60,26 @@ Delay class >> delaySchedulerClass: aSchedulerClass [
 	self inform: 'Delay scheduler set to ' , aSchedulerClass printString
 ]
 
+{ #category : 'instance creation' }
+Delay class >> forDays: aNumber [
+	"Return a new Delay for the given number of Days"
+	
+	^ self forMilliseconds: aNumber * 1000 * 60 * 60 * 24
```

Comment:
Hmm, shouldn't this use the constants in `ChronologyConstants`? It makes sense that `Delay` depends on that

Reply:
Yes it would be better.  Now we can do it in a second turn. Because it would be good to have it run and then create extra package.